RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate getSlashCommands to TS

## Proposed changes

Migrate getSlashCommands to TS.

CHANGELOG:
> New interface which extends `ISlashCommands` for the `commands.list` response from server
> Migrate getSlashCommands to TS
> Small spacing misplaced on `IRocketChat`
## Issue(s)	


## How to test or reproduce

This function runs on the app startup, so just ensure that slash commands load properly
## Screenshots

## Types of changes




## Checklist



## Further comments
